main.py

```diff
@@ -43,75 +50,51 @@ def load_assets(self):
         self.font = import_font(30, 'font/LycheeSoda.ttf')
 
     def run(self):
-        while self.running:
+        while self.game_state[-1] == 'game':
```

Comment:
Why change?

Reply:
i did that so just changing the head of the state stack will close the loop this also helped unifying the exit point in the code (could help in saving etc)